src/components/admin/RichTextEditor.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import RichTextEditor from 'react-rte';
+
+export default class MyEditor extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: RichTextEditor.createEmptyValue(),
+    };
+    this.onChange = this.onChange.bind(this);
+  }
+
+  componentDidMount() {
+    const { defaultValue } = this.props;
+    if (defaultValue) {
+      this.setState({ value: RichTextEditor.createValueFromString(defaultValue, 'html') });
```

Comment:
Could you explain why you pass 'html' to the createValueFromString function

Reply:
They support multiple text formats. html, markdown, and I think a few others. So 'html' tells them to render the input string as html.